lib/install/config/development.js

```diff
@@ -20,6 +21,8 @@ module.exports = merge(config, {
   plugins: [
     new webpack.LoaderOptionsPlugin({
       debug: true
-    })
+    }),
+
+    new BundleTracker({ path: __dirname, filename: '../../tmp/webpack-stats.json' })
```

Comment:
You should use `path.join()` for windows :)
`filename: path.join('..', '..', 'tmp', 'webpack-stats.json')`